vagrant/provisioning/setup-minion.sh

```diff
@@ -183,16 +155,17 @@ else
     ovn_sb="$PROTOCOL://$MASTER1:6642"
 fi
 
+TOKEN=`sudo cat /vagrant/token`
```

Comment:
I didn't check too closely, but are we sure the token will have been written by the master here?  Do we need to wait for it to show up if the minion somehow starts sooner?

Reply:
The documentation for vagrant asks to start master first. So I think we are okay here.